usaspending_api/disaster/v2/views/federal_account/spending.py

```diff
@@ -175,14 +175,19 @@ def total_queryset(self):
             ),
             "total_budgetary_resources": Coalesce(
                 Subquery(
-                    GTASSF133Balances.objects.filter(
+                    latest_gtas_of_each_year_queryset()
+                    .filter(
                         disaster_emergency_fund_code__in=self.def_codes,
-                        fiscal_period=self.latest_reporting_period["submission_fiscal_month"],
-                        fiscal_year=self.latest_reporting_period["submission_fiscal_year"],
                         treasury_account_identifier=OuterRef("treasury_account"),
                     )
-                    .annotate(amount=Func("total_budgetary_resources_cpe", function="Sum"))
-                    .values("amount"),
+                    .annotate(
+                        amount=Func("total_budgetary_resources_cpe", function="Sum"),
+                        unobligated_balance=Func(
+                            "budget_authority_unobligated_balance_brought_forward_cpe", function="Sum"
+                        ),
+                    )
+                    .annotate(total_budget_authority=F("amount") - F("unobligated_balance"))
```

Comment:
Saw your comment about not liking the generated SQL. Curious why a single annotation wouldn't be acceptable.
```suggestion
                    .annotate(
                        total_budget_authority=Func("total_budgetary_resources_cpe", function="Sum")
                        - Func("budget_authority_unobligated_balance_brought_forward_cpe", function="Sum")
                    )
```

Reply:
Regardless of that change the SQL is the same with the select containing:

>coalesce((select (Sum(U0."total_budgetary_resources_cpe") - Sum(U0."budget_authority_unobligated_balance_brought_forward_cpe")) as "total_budget_authority" from "gtas_sf133_balances" U0 where (((U0."fiscal_year" = 2021 and U0."fiscal_period" = 2) or (U0."fiscal_year" = 2020 and U0."fiscal_period" = 12)) and U0."disaster_emergency_fund_code" in (L, M, N, O, P) and U0."treasury_account_identifier" = ("financial_accounts_by_program_activity_object_class"."treasury_account_id"))), 0) as "total_budgetary_resources"

My complaint is that as a whole `self.total_queryset` is a pretty nasty looking bit of SQL. This really doesn't change it that much.